openupgrade_scripts/scripts/hr_expense/15.0.2.0/pre-migration.py

```diff
@@ -0,0 +1,57 @@
+from openupgradelib import openupgrade
+
+
+def _compute_payment_state(env):
+    if not openupgrade.column_exists(
+        env.cr,
+        "hr_expense_sheet",
+        "payment_state",
+    ):
+        openupgrade.logged_query(
+            env.cr,
+            """
+            ALTER TABLE hr_expense_sheet
+            ADD COLUMN payment_state varchar
+            """,
+        )
```

Comment:
@daonamutc1 sử dụng `IF NOT EXISTS` thì sẽ ngắn hơn, không phải mất đến 2 query anh nhé
```suggestion
      openupgrade.logged_query(
          env.cr,
          """
          ALTER TABLE hr_expense_sheet
          ADD COLUMN IF NOT EXISTS payment_state varchar
          """,
      )
```